### DISCUSSION TITLE
Some Lambdas with strange names were automatically created by Serverless

### DISCUSSION THREAD

**Discussion by cjnoname** [serverless/serverless#12231] [Q&A]
Hi,

We have found that there're some Lambdas created by Serverless with strange names. DevOps teams report that there are security concern with these Lambdas as they have the "PassRole" IAM permission etc. 

They have name ends with "custom-resource-existing-cup" or ends with "cognito-messagd3845b904085bd2b2697a50918c11951".

Is there a way to delete them after the deployment?

Anyone can help?

Thanks,
CJ

**Comment by pgrzesik** [comment]
Hey 👋 In some cases, the functionality supported by Serverless Framework requires use of CloudFormation Custom Resources (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-custom-resources.html) that are backed by these Lambda functions. You cannot delete them as it would break some of the existing functionality in your app that depends on them.
